openmc/geometry.py

```diff
@@ -65,14 +63,22 @@ def get_offset(self, path, filter_offset):
 
         """
 
+        # Find the distribcell index of the cell.
+        cells = self.get_all_cells()
+        if path[-1] in cells:
+            distribcell_ind = cells[path[-1]].distribcell_ind
```

Comment:
Very good point about the overloaded use of the term "offset" - I think we should cogitate on the terms `offset` and `distribcell_ind` with the other developers before merging this in.